addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -624,6 +648,12 @@ function factory(dependencies) {
             compute: '_computeHasGuestNameChanged',
             readonly: true,
         }),
+        /**
+         * Determine whether description should display on top bar.
+         */
+        hasThreadViewTopbarDescription: attr({
+            compute: '_computeHasThreadViewTopbarDescription',
+        }),
```

Comment:
Just rename to `hasDescription` (more concise, as `ThreadViewTopbar` is the name of current model, so it's redundant to put in field name).

Reply:
@alexkuhn Done.